frontend/packages/knative-plugin/src/components/add/import-types.ts

```diff
@@ -1,10 +1,11 @@
 import { LimitsData } from '@console/dev-console/src/components/import/import-types';
 
 export enum EventSources {
-  CronJobSource = 'CronJobSource',
-  ContainerSource = 'ContainerSource',
   ApiServerSource = 'ApiServerSource',
+  ContainerSource = 'ContainerSource',
+  CronJobSource = 'CronJobSource',
   KafkaSource = 'KafkaSource',
+  PingSource = 'PingSource',
   CamelkSource = 'CamelSource',
   SinkBinding = 'SinkBinding',
 }
```

Comment:
Are values in this enum kind of the respective EventSource? If yes, we should be using their respective Model.kind here.

Reply:
yes, but then may have to use normal Object as in enum's `computed values are not permitted in an enum with string valued members`

Reply:
have made it a `const` and used model.kind